assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -149,7 +150,7 @@ const Cart = ( { attributes } ) => {
 					currency={ totalsCurrency }
 					values={ cartTotals }
 				/>
-				<ExperimentalOrderMeta.Slot />
+				<ExperimentalOrderMeta.Slot cart={ cart } />
```

Comment:
I don't think we should expose `cart.receiveCart` on this property.

Reply:
yeah I need to shake down that value before returning it.